akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -155,6 +155,12 @@ object ShardRegion {
   @SerialVersionUID(1L) final case object GracefulShutdown extends ShardRegionCommand
 
   /**
+   * We must be sure that a shard is initialized before to start send messages to it.
```

Comment:
as mentioned elsewhere: Sometimes persistent shard could be failure during a replay process due to temporary journal unavailability. The user messages which sent to such shard just after creation, but before a failure would be lost. This code removes that problem.